src/osdatahub/utils.py

```diff
@@ -193,4 +197,15 @@ def validate_in_range(value: float, minimum: float, maximum: float) -> float:
     """
     if value < minimum or value > maximum:
         raise ValueError(f"Value should be between {minimum} and {maximum}, got {value}.")
-    return value
\ No newline at end of file
+    return value
+
+
+def is_new_api(response: Union[dict, GrowList]) -> bool:
+    if isinstance(response, GrowList) and len(response) > 0:
```

Comment:
Would it be possible to add a quick comment explaining why these criteria indicate a new api key has been used (mostly for my own interest)? :)

Reply:
Done!